CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 
 ### Bug fixes
 
+* [#5623](https://github.com/bbatsov/rubocop/pull/5623): Fix `Bundler/OrderedGems` when a group includes duplicate gems. ([@colorbox][])
```

Comment:
The changelog entry should be moved, as a new release was cut a few days ago.

Reply:
Thank you. 🙏 Okay, I'l fix it.